scripts/create_mb_refstate

```diff
@@ -124,6 +126,12 @@ def get_mok(events):
 
 
 def main():
+    tpm = tpm_main.tpm()
+
+    if Version(tpm.tools_version) < Version("4.2"):
+        print(f"tpm2_eventlog is not available (min tpm2-tools version is 4.2)")
+        return
```

Comment:
Should it not signalize a retcode != 0? (`sys.exit(1)`, for example)

Reply:
Yes, you are right, I'll fix these